src/display/editor/editor.js

```diff
@@ -140,6 +140,26 @@ class AnnotationEditor {
     return [];
   }
 
+  /**
+   * Check if this kind of editor is able to handle the given mime type for
+   * pasting.
+   * @param {string} mime
+   * @returns {boolean}
+   */
+  static isHandlingMimeForPasting(_mime) {
+    return false;
+  }
+
+  /**
+   * Extract the data from the clipboard item and delegate the creation of the
+   * editor to the parent.
+   * @param {DataTransferItem} item
+   * @param {AnnotationEditorLayer} parent
+   */
+  static paste(item, parent) {
+    unreachable("Must be able to paste some data.");
```

Comment:
Nit: Maybe just shorten this slightly, more inline with other existing code?
```suggestion
    unreachable("Not implemented");
```